### ISSUE TITLE
ensure all data (speeds/etc) saved in pset

### ISSUE THREAD

**Issue by tehn** [tehn/mlr#24]

**Comment by tehn** [comment]
this actually might mean that track speed settings aren't in the param menu

**Comment by sonoCircuits** [comment]
I wrote a "fix" that has direction and speed in the param menu. what other params were you thinking of?

**Comment by tehn** [comment]
yes that'd work! i'm wondering if it introduces some issues to have them in the params, ie, midi-mapping or altering them through the menu means that settings aren't properly updated? (just to say, for consistency this would need to have extra glue coded all around...)

**Comment by sonoCircuits** [comment]
the way I've set it up is that all updating is handled in the params menu. grid key presses point to the according params:set.
I can send you a PR to check my code if you like?

**Comment by tehn** [comment]
sure, happy to take a look!
